runtime/compiler/trj9/Makefile

```diff
@@ -37,6 +37,19 @@ ifeq ($(SPEC),linux_arm)
     $(warning ARM SPEC detected)
 endif
 
+# During openJDK build, SPEC arrives as 
+# /root/openj9-openjdk-jdk9/build/linux-arm-normal-server-release/spec.gmk
+# and PLATFORM is not set, so the above doesn't work.
+ifdef OPENJ9_CC_PREFIX
+  PLATFORM=arm-linux-gcc-cross
```

Comment:
Similarly the platform shouldn't be assumed, you can update runtime/compiler/makefile.ftl to set it